product_tier_validation/data/product_state_data.xml

```diff
@@ -1,8 +1,7 @@
 <odoo noupdate="1">
-    <record id="product_state.product_state_sellable" model="product.state">
```

Comment:
is `confirmed` a state this module expects?

Reply:
I could not found any place that uses product code `confirmed` (https://github.com/OCA/product-attribute/search?q=confirmed&type=code). AFAIK, `code` is used by py code or xml. With existing dbs those have the module installed, join between `product.template` and `product.state` is based on id, not on code. In theory, we should not change code.